Unix/build.mak

```diff
@@ -237,7 +237,7 @@ distclean:
 
 ifdef BUILD_TESTS
 tests:
-	( LD_LIBRARY_PATH=$(LIBDIR); export LD_LIBRARY_PATH; DYLD_LIBRARY_PATH=$(LIBDIR); export DYLD_LIBRARY_PATH; $(BINDIR)/nits -file:$(TMPDIR)/nitsargs.txt )
+	sudo sh -c "( LD_LIBRARY_PATH=$(LIBDIR); export LD_LIBRARY_PATH; DYLD_LIBRARY_PATH=$(LIBDIR); export DYLD_LIBRARY_PATH; $(BINDIR)/nits -file:$(TMPDIR)/nitsargs.txt )"
```

Comment:
I assume there's a reason for sudo here, and in other places.  Nevertheless, this bothers me...